### ISSUE TITLE
Django 2.1.x and Python 3.6 compiant

### ISSUE THREAD

**Issue by ghost** [makinacorpus/django-tracking-fields#4]
I made a fork and started some work but in fact, it is not limited to the adding of "on_delete" on each ForeignKey and OnToOne, there is also this when I run the unit tests

```
$ ./manage.py test
System check identified some issues:

WARNINGS:
tracking_fields.Human.pets: (fields.W340) null has no effect on ManyToManyField.

System check identified 1 issue (0 silenced).
E
======================================================================
ERROR: tracking_fields.tests (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: tracking_fields.tests
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/unittest/loader.py", line 462, in _find_test_path
    package = self._get_module_from_name(name)
  File "/usr/local/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/home/foxmask/DjangoVirtualEnv/tracking-fields/django-tracking-fields/tracking_fields/tests/__init__.py", line 1, in <module>
    from . import tests
  File "/home/foxmask/DjangoVirtualEnv/tracking-fields/django-tracking-fields/tracking_fields/tests/tests.py", line 19, in <module>
    from tracking_fields.tests.models import Human, Pet, House
  File "/home/foxmask/DjangoVirtualEnv/tracking-fields/django-tracking-fields/tracking_fields/tests/models.py", line 33, in <module>
    class House(models.Model):
  File "/home/foxmask/DjangoVirtualEnv/tracking-fields/django-tracking-fields/tracking_fields/decorators.py", line 134, in inner
    _track_class(cls, fields)
  File "/home/foxmask/DjangoVirtualEnv/tracking-fields/django-tracking-fields/tracking_fields/decorators.py", line 96, in _track_class
    _track_class_field(cls, field)
  File "/home/foxmask/DjangoVirtualEnv/tracking-fields/django-tracking-fields/tracking_fields/decorators.py", line 77, in _track_class_field
    _track_class_related_field(cls, field)
  File "/home/foxmask/DjangoVirtualEnv/tracking-fields/django-tracking-fields/tracking_fields/decorators.py", line 41, in _track_class_related_field
    related_cls = field_obj.rel.to
AttributeError: 'OneToOneField' object has no attribute 'rel'
```
so as it's going to deeper for me, I removed the fork and let you manage that piece of code I don't control :)

**Comment by ghost** [comment]
in the logs of travis I saw a log that made me think the project also need an `apps.py`

**Comment by Gagaro** [comment]
released in 1.1.0
